package.json

```diff
@@ -78,10 +78,14 @@
   },
   "scripts": {
     "dev": ". ./scripts/populate-git-env.sh && next dev -p 3100",
+    "dev:win": "./scripts/populate-git-env.sh && next dev -p 3100",
     "debug": ". ./scripts/populate-git-env.sh && WDYR=1 NODE_OPTIONS='--inspect' next dev -p 3100",
+    "debug:win": "./scripts/populate-git-env.sh && WDYR=1 NODE_OPTIONS='--inspect' next dev -p 3100",
     "start": "next start -p 3100",
     "build": ". ./scripts/populate-git-env.sh && next build",
+    "build:win": "./scripts/populate-git-env.sh && next build",
     "export": ". ./scripts/populate-git-env.sh && next export",
+    "export:win": "./scripts/populate-git-env.sh && next export",
```

Comment:
These two commands are not really equivalent to each other, since you aren't running the shell script with with `.` (or `source`), the exported environment variables aren't actually being included in the context of `next *`.

I think we either find a way to make the definition of these env vars work on windows, or we just drop the running of those scripts on windows so that we don't mislead a developer into believing they are actually doing something.